howso/train.amlg

```diff
@@ -1,6 +1,20 @@
+;Contains methods for training (input of data into the trainee).
 (null
 
-	;records an entire session in one call, does not do any filtering.
+	;train the passed in cases, filtering out cases that match optionally passed in ablation parameters
+	; returns a response object in the following format:
+	;	(associate
+	;		"success" (true)/(false)
+	;		"num_trained" num_trained_cases
+	;		"ablated_indices" (list of ablated case indices)
+	;		"status" "status output message"
+	;		"message" "if failed to succeed, the error/reason message will be here"
```

Comment:
it doesn't seem to me that "message" is a key in the returned assoc. Looks like if there is a "message", it gets stuffed in as an "error"